openfisca/api_config.ini

```diff
@@ -0,0 +1,49 @@
+# OpenFisca-Web-API - Development environment configuration
+#
+# The %(here)s variable will be replaced with the parent directory of this file.
+
+[DEFAULT]
+debug = true
+
+[server:main]
+use = egg:Paste#http
+host = 0.0.0.0
+port = %PORT%
+
+[app:main]
+use = egg:OpenFisca-Web-API
+country_package = openfisca_france
+log_level = DEBUG
+reforms =
+    openfisca_france.reforms.aides_cd93.aides_cd93
+extensions=
+    openfisca_paris
```

Comment:
Only historical reasons:
- At first, only `reforms` were available. The API was pretty awful but we dealt with it to build POCS with Paris and CD93.
- When we started to work more seriously with Paris, we built the extension mechanism and got rid of the Paris reform
- We never paid the relatively small cost of turing `cd93` into an extension since as of today it a dead project.

Do you think consistency is worth paying the additional cost (I guess 1-3 hours) ?